src/formats/geotiff/GeoTiffReader.js

```diff
@@ -888,22 +888,6 @@ define([
             }
         }
 
-        // Get metadata from GeoKeys. Internal use only.
-        GeoTiffReader.prototype.getMetadataFromGeoKeys = function () {
-            for (var i = 0; i < this.geoKeys.length; i++) {
-                var keyAsString = GeoTiffUtil.getTagValueAsString(GeoTiffConstants.Key, this.geoKeys[i].keyId);
```

Comment:
This function indeed looks like obsolete code. In addition, please check whether `GeoTiffUtil.getTagValueAsString` is still needed.

Reply:
I reviewed the code and I removed the unused functions:
* getTagValueAsString
* clampColorSampleForElevation
* getMinMaxGeotiffSamples